onnxscript/rewriter/pattern.py

```diff
@@ -1830,7 +1830,7 @@ def report(self) -> None:
             print(f"Rule: {rule}")
             print(f"Best score: {matches[0].score()}")
             for match in matches:
-                print(f"Status: {match.status}")
+                print(f"Status: {match.status.name}")
```

Comment:
We can make status a String Enum

Reply:
It is also used as an int in calculating a "score" (for the "best" match).